modules/percolator/src/test/java/org/elasticsearch/percolator/PercolateQueryBuilderTests.java

```diff
@@ -354,7 +332,7 @@ public void testSettingNameWhileRewritingWhenDocumentSupplierAndSourceNotNull()
         Supplier<BytesReference> supplier = () -> new BytesArray("{\"test\": \"test\"}");
         String testName = "name1";
         QueryShardContext shardContext = createShardContext();
-        PercolateQueryBuilder percolateQueryBuilder = new PercolateQueryBuilder(queryField, null, supplier);
+        PercolateQueryBuilder percolateQueryBuilder = new PercolateQueryBuilder(queryField, supplier::get);
```

Comment:
Super small comment, could this just be `new PercolateQueryBuilder(queryField, supplier)`?